appveyor.yml

```diff
@@ -1,4 +1,3 @@
-version: 3.5.{build}
```

Comment:
This was inconsistent with the rest of the versions, and as far as I can see, overridden by cake anyway? Maybe I missed something.